scripts/release/mule/Makefile.mule

```diff
@@ -46,30 +46,19 @@ ci-build: ci-setup buildsrc gen
 		 NO_BUILD=True VARIATIONS=$(OS_TYPE)/$(ARCH) \
 		 scripts/build_packages.sh $(OS_TYPE)/$(ARCH)
 
-package-deb:
-	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/deb/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
+# Builds targets from the sub-directories of ./scripts/release/mule/, such as `mule-package`, `mule-sign`, `mule-test`.
+# https://scene-si.org/2019/12/04/make-dynamic-makefile-targets/
+mule = $(shell ls -d scripts/release/mule/*/ | awk 'BEGIN { FS="/" ; OFS="-" } { print $$3, $$4 }')
```

Comment:
what does this do?

Reply:
By that I mean the awk portion.

Reply:
It parses the output of `ls` into the target into the target prefixes.
```
$ ls -d scripts/release/mule/*/
scripts/release/mule/package/  scripts/release/mule/sign/  scripts/release/mule/test/

$ ls -d scripts/release/mule/*/ | awk 'BEGIN { FS="/" ; OFS="-" } { print $3, $4 }'
mule-package
mule-sign
mule-test
```

Reply:
The article this is based on that I linked to in the comments in the code uses `sed`, but I find `awk` to be easier to use.

Reply:
Is this a make target? Shouldn't that be a `:`?

Reply:
No, that's not a target.